examples/build.sbt

```diff
@@ -6,47 +6,63 @@ benchmarks ++= Seq(
   suiteForEncoding_examples(examplesSpecs)
 )
 
+// (folder, spec, init, inv, length)
 lazy val examplesSpecs = Seq(
-  ("MC4_FALSE_FALSE.tla", "Consistency"),
-  ("MC4_TRUE_TRUE.tla", "Consistency"),
-  ("MC10_FALSE_FALSE.tla", "Consistency"),
-  ("MC10_TRUE_TRUE.tla", "Consistency"),
-  ("MC20_FALSE_FALSE.tla", "Consistency"),
-  ("MC20_TRUE_TRUE.tla", "Consistency")
+  ("2PCwithBTM", "MC4_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC4_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC10_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC10_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC20_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC20_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("aba_asyn_byz", "MC4.tla", "Init", "", "10"),
+  ("aba_asyn_byz", "MC10.tla", "Init", "", "10"),
+  ("aba_asyn_byz", "MC20.tla", "Init", "", "10"),
+  ("bakery", "MC3.tla", "Init", "MutualExclusion", "10"),
+  ("bakery", "MC3.tla", "Init", "Inv", "0"),
+  ("bakery", "MC3.tla", "Inv", "Inv", "1"),
+  ("changRoberts", "MC4.tla", "Init", "Correctness", "10"),
+  ("changRoberts", "MC10.tla", "Init", "Correctness", "10"),
+  ("changRoberts", "MC20.tla", "Init", "Correctness", "10"),
+  ("paxos", "MC3.tla", "Init", "V!OneValuePerBallot", "13"),
+  ("readersWriters", "MC4.tla", "Init", "Safety", "10"),
+  ("readersWriters", "MC10.tla", "Init", "Safety", "10"),
+  ("readersWriters", "MC20.tla", "Init", "Safety", "10")
 )
 
-def suiteForEncoding_examples(specs: Seq[(String, String)]) = {
+def suiteForEncoding_examples(specs: Seq[(String, String, String, String, String)]) = {
```

Comment:
It occurs to me that using a simple object to have named fields here might make this more maintainable and easier to read. A five-tuple is pretty big! :)

Perhaps something like

```scala
case class Spec(folder: String, fileName: String, init: String = "Init", inv: String = "Inv", length: Int)
```

WDYT?

Reply:
I like the idea, but I am having trouble importing `Spec` into `build.sbt`. I looked at `BenchExecDsl` but couldn't get anything out of it. Is there something specific to sbt that I have to do?